charts/core/src/Header/Header.styles.ts

```diff
@@ -0,0 +1,28 @@
+import { css } from '@leafygreen-ui/emotion';
+import { Theme } from '@leafygreen-ui/lib';
+import {
+  color,
+  InteractionState,
+  spacing,
+  Variant,
+} from '@leafygreen-ui/tokens';
+
+export const getContainerStyles = (theme: Theme, showDivider?: boolean) => css`
```

Comment:
A tip i got from Adam was to use 
```
cx(css`...`, {[css`...`]:showDivider})
```
instead of 
```
css`
...
${showDivider && ``}
`
```
`cx` is better for conditional classes

Reply:
Cool! Will do!